jupyterhub_traefik_proxy/etcd.py

```diff
@@ -54,17 +56,35 @@ def _default_client(self):
     etcd_traefik_prefix = Unicode(
         "/traefik/",
         config=True,
-        help="""the etcd key prefix for traefik configuration""",
+        help="""the etcd key prefix for traefik static configuration""",
     )
 
     etcd_jupyterhub_prefix = Unicode(
         "/jupyterhub/",
         config=True,
-        help="""the etcd key prefix for traefik configuration""",
+        help="""the etcd key prefix for traefik dynamic configuration""",
     )
 
+    traefik_api_url = Unicode(
+        "http://127.0.0.1:8099",
+        config=True,
+        help="""traefik authenticated api endpoint url""",
+    )
+
+    traefik_api_password = Unicode()
```

Comment:
username and password here should have `config=True` and `help` so they can come from configuration. I think this is the last bit for this PR.

Reply:
Thanks @minrk! I've just added these changes.